target/linux/ramips/base-files/lib/upgrade/platform.sh

```diff
@@ -256,6 +256,7 @@ platform_check_image() {
 	tplink,c50-v3|\
 	tplink,tl-mr3420-v5|\
 	tplink,tl-wr902ac-v3|\
+	tl-wa801nd-v5|\
```

Comment:
Drop it from this file and use the devicetree compatible string `tplink,tl-wa801nd-v5`as userspace boardname instead of `tl-wa801nd-v5`.